app/models/planning_application.rb

```diff
@@ -458,9 +458,23 @@ def planning_history_enabled?
     ENV.fetch("PLANNING_HISTORY_ENABLED", "false") == "true"
   end
 
+  def rejected_assessment_detail(category:)
+    return unless recommendation&.rejected?
+
+    assessment_details.where(
+      category: category,
+      status: :review_complete,
+      review_status: :rejected
+    ).last
+  end
+
   AssessmentDetail.categories.each_key do |category|
     define_method(category) do
-      assessment_details.find_by(category: category)
+      assessment_details.where(category: category).first
```

Comment:
Ensure this returns the most recent record (they're ordered by `created_at_desc`)